isos_post.py

```diff
@@ -112,8 +117,12 @@ def get_job_name(host, job_id):
     print("Failed Jobs:")
     for job in o_json['failed']:
         print("  %s/t%d - Name:%s MSG:%s" % (args.host, job['job_id'], get_job_name(args.host, job['job_id']), job['error_messages']))
+        if (args.priority is not None):
```

Comment:
if we in failed jobs section no point create more mess by setting priorities . please remove it from here

Reply:
The problem is, if we `--nostartafter` we always fail, cause of this. And then we would not set the priority even if it was successful triggered.

Reply:
so we need to detect that --nostartafter is used and error message contains START_AFTER_TEST

Reply:
Or we fix openqa-client to allow clearing variables :)... I think your idea goes faster

Reply:
but still we can give it a try ( fix openqa-client ) don't think should be hard ... at least from coding perspective :\